### PR TITLE
Reorganize community discussion section

### LINKED ISSUE #799: Re-organize Community Discussion section
The [Instructor Development section](https://docs.carpentries.org/topic_folders/instructor_development/index.html) currently has two sections on Community Discussions.  This could be re-organized to streamline information sharing and remove information that is intended for the internal Core Team.  A summary of the current and proposed structure is below.  I will be discussing this with the Community Development Team.  

# Current Structure
## Community Discussions

* Introduction; definitions of pre-post discussion/themed discussion/community conversation
* Terminology: Community discussion; Discussion session host
* Motivation
* Who can host
   * Discussion host onboarding
* Signing up to Host
* Host expectations
* Co-host expectations
* Checklists
* FAQ

## Community Discussions Workflow

* Scheduling Pre and Post Workshop Community Discussions
* Preparing to Host ....
* Community Discussion Wrapup

# Proposed structure

## Community Discussions

* Introduction; definitions of pre-post discussion/themed discussion/community conversation **_Keep this_**
* Terminology: Community discussion; Discussion session host **_Remove this_**
* Motivation **_Keep this_**
* Who can host  **_Keep this pending CDT plans_**
   * Discussion host onboarding
* Signing up to Host **_Keep this section but link to information shared by instructor training demo scheduling_**
* Host expectations, Co-host expectations, Checklists: **_Keep this but combine information_**
* FAQ **_Keep this (with review)_**

## Community Discussions Workflow - **_remove this section_**

* Scheduling... **_Remove, as this is internal workflow_**
* Preparing to host - **_add to host expectations/checklist_**
* Wrap up - **_add to host expectations/checklist_**

### PR DESCRIPTION
**Pull request by maneesha**
Fixes #799

### CODE REVIEW

**Comment by maneesha** [comment]
This PR reorganizes the [Community Discussions](https://docs.carpentries.org/topic_folders/instructor_development/community_discussions.html) section of the Handbook. It also moves the section on scheduling outside of community discussions, since the same workflow is used for Instructor Training. Once the PR is merged, the Instructor Training section will also link to this new scheduling section.

[Preview community discussion section](https://github.com/carpentries/docs.carpentries.org/blob/5fab45ee6d3e95d5a9670c015fdaad33febe8dcb/topic_folders/instructor_development/community_discussions.md)
[Preview scheduling section](https://github.com/carpentries/docs.carpentries.org/blob/5fab45ee6d3e95d5a9670c015fdaad33febe8dcb/topic_folders/communications/guides/community_events.md)

@acrall  You can use the preview links above to see roughly what the pages will look like so you can review and approve.  Seeing what they look like for real in the handbook will require building the handbook locally.  I can set some time to screen share with you if you want to see that before merging.

File: topic_folders/instructor_development/community_discussions.md [hunk 1]
```diff
@@ -50,34 +45,13 @@ Interested in becoming a session host? Not sure how or what would be required of
...
-Responses<br>due by      |30 Nov              |28 Feb               |31 May               |31 Aug               |
-Calendar<br>published by |7 Dec               |7 Mar                |7 Jun                |7 Sept               |
-
-Once Discussion Hosts have scheduled themselves in for a Community Discussion session via Calendly, The Carpentries Core Team will ensure that these events appear on the [Etherpad](https://pad.carpentries.org/community-discussions) and on the [Community Calendar](https://carpentries.org/community/#community-events).  
+Discussion sessions are scheduled on a quaterly basis [as described here](../communications/guides/community_events.md). This will allow The Carpentries Community Development Team to support your recruitment initiatives and help publicise your event. In the rare instance that you need to schedule such sessions outside of a quarter please complete [this form](https://docs.google.com/forms/d/e/1FAIpQLSen9_axxQ3_0FN5HjL7cyot9RzTdIGpOU16Wr1eatZblsfU7w/viewform). Your session is not confirmed until you hear from a member of The Carpentries Community Development Team.
```
Comment (acrall) [inline topic_folders/instructor_development/community_discussions.md:50]: @maneesha "quaterly" should be "quarterly"

File: topic_folders/instructor_development/community_discussions.md [hunk 2]
```diff
@@ -88,7 +62,7 @@ We recognise that even after scheduling their own events, Discussion Hosts may h
 * Encouraging attendees to create issues or pull requests to correct problems.
```
Comment (acrall) [inline topic_folders/instructor_development/community_discussions.md:62]: Change "encouraging" to "encourage"

**Review by acrall — APPROVED** [review]
@maneesha I just made a couple of minor edits although I'm not sure I submitted them correctly so let me know. Thanks!

**Pull request merged**